crates/biome_css_parser/src/lexer/mod.rs

```diff
@@ -21,6 +21,12 @@ pub enum CssLexContext {
     /// Applied when lexing CSS pseudo nth selectors.
     /// Distinct '-' from identifiers and '+' from numbers.
     PseudoNthSelector,
+
+    /// Applied when lexing CSS url function.
+    /// Doesn't skip whitespace trivia for a combinator.
```

Comment:
`/// Doesn't skip whitespace trivia for a combinator.` 
I guess that we need to remove it because we don't have any combinator in css url functions.